spring-cloud-dataflow-composed-task-runner/src/test/java/org/springframework/cloud/dataflow/composedtaskrunner/ComposedRunnerVisitorTests.java

```diff
@@ -397,23 +397,16 @@ private void setupContextForGraph(String[] args) throws RuntimeException{
 				SimpleTaskAutoConfiguration.class}, args);
 	}
 
-	private Collection<StepExecution> getStepExecutions() {
-		return getStepExecutions(false);
-	}
 
-	private Collection<StepExecution> getStepExecutions(boolean isCTR) {
+	private Collection<StepExecution> getStepExecutions() {
 		JobExplorer jobExplorer = this.applicationContext.getBean(JobExplorer.class);
 		List<JobInstance> jobInstances = jobExplorer.findJobInstancesByJobName("job", 0, 1);
 		assertThat(jobInstances).hasSize(1);
 		JobInstance jobInstance = jobInstances.get(0);
 		List<JobExecution> jobExecutions = jobExplorer.getJobExecutions(jobInstance);
 		assertThat(jobExecutions).hasSize(1);
 		JobExecution jobExecution = jobExecutions.get(0);
-		if(isCTR) {
-			assertThat(jobExecution.getJobParameters().getParameters().get("ctr.id")).isNotNull();
-		} else {
-			assertThat(jobExecution.getJobParameters().getParameters()).containsEntry("run.id", new JobParameter(1L, Long.class));
```

Comment:
Is this related to the deprecation or was it just not being used?

Reply:
Great Question!   This is a test to make sure that either the "incrementer" id was set by the builder for CTR.  In batch the `incrementer` is an autogenerated number or value that will make each new execution unique.

Reply:
Got it. 👍🏻